src/main/java/org/sert2521/gamename/OI.kt

```diff
@@ -1,11 +1,12 @@
 package org.sert2521.gamename
 
+import org.strongback.components.ui.FlightStick
 import org.strongback.hardware.Hardware
 
 /**
  * This object defines the operator interface controls, such as joysticks, buttons and switches.
  */
 object OI {
-    val left = Hardware.HumanInterfaceDevices.logitechAttack3D(LEFT_STICK_PORT)
-    val right = Hardware.HumanInterfaceDevices.logitechAttack3D(RIGHT_STICK_PORT)
+    val left: FlightStick = Hardware.HumanInterfaceDevices.logitechAttack3D(LEFT_STICK_PORT)
+    val right: FlightStick = Hardware.HumanInterfaceDevices.logitechAttack3D(RIGHT_STICK_PORT)
```

Comment:
Kotlin can't figure if these are going to be null or not (though we know they're non-null). So we explicitly define them as non-null.